client/server.go

```diff
@@ -314,11 +333,16 @@ func newCqlServerConnection(
 		codec:       codec,
 		credentials: credentials,
 		idleTimeout: idleTimeout,
+		handlers:    handlers,
+		handlerCtx:  make([]RequestHandlerContext, len(handlers)),
```

Comment:
Same comment as before, this `make` call is a bit confusing because we expect it to be used with arrays, maps and channels but we don't know right away what `RequestHandlerContext` is without looking it up.